openquake/hazardlib/gsim/campbell_bozorgnia_2003_world.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2023 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`CampbellBozorgnia2003`
+                class:`CampbellBozorgnia2003Vertical`.
+"""
+import numpy as np
+
+from openquake.hazardlib.gsim.base import GMPE, CoeffsTable
+from openquake.hazardlib import const
+from openquake.hazardlib.imt import PGA, SA
+
+
+def _get_mean(C, mag, rake, dip, rrup, rjb, vs30):
+    """
+    Return mean value (eq. 1, page 319).
+    """
+    f1 = _compute_magnitude_scaling(C, mag)
+    f2 = _compute_distance_scaling(C, mag, rrup, vs30)
+    f3 = _compute_faulting_mechanism(C, rake, dip)
+    f4 = _compute_far_source_soil_effect(C, vs30)
+    f5 = _compute_hanging_wall_effect(C, rjb, rrup, dip, mag, rake, vs30)
+    return C['c1'] + f1 + C['c4'] * np.log(np.sqrt(f2)) + f3 + f4 + f5
+
+
+def _compute_magnitude_scaling(C, mag):
```

Comment:
This function seems a repetition of https://github.com/gem/oq-engine/blob/master/openquake/hazardlib/gsim/campbell_bozorgnia_2003.py#L41 Another possibility would have been to import this function rather than repeating it